.circleci/config.yml

```diff
@@ -0,0 +1,159 @@
+version: 2.1
+
+jobs:
+  ruby_style: &bazel_defaults
+    working_directory: /home/circleci/repo
+    resource_class: medium
+    docker:
+      - image: bazelruby/ruby-2.6.5
```

Comment:
why not use a default circle image?

Reply:
Just to save time for the build — less things to install.